Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -10514,7 +10488,7 @@ - (void)useSelectionForFindForWebView:(id)sender
 - (void)_findSelectedForWebView:(id)sender
 {
     [self useSelectionForFindForWebView:sender];
-    [self findForWebView:sender];
+    [self.webView find:nil];
```

Comment:
Are we intentionally dropping the `sender` here?

Reply:
Not intentional, I put it back.